### PR TITLE
Decade of Data Blog Post

### PR DESCRIPTION
**Pull request by braunjj**
Add Nick's Series A announcement blog post – A Decade of Data

### CODE REVIEW

**Review by bengotow — APPROVED** [review]
Not sure if this needs a stamp, looks good! 🛳

File: pages/blog/decade-of-data.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+date: '2021-11-16'
+authors: ['schrockn']
+tags: ['Blog Post']
+coverImage: '/posts/decade-of-data/dagster+elementl.gif'
+
+---
+
+Dagster has been built out in the open, but we’ve been quietly building Elementl, the company behind it, along the way.
+
+Today, I’m excited to announce that Elementl raised a $14M Series A led by Mike Volpi of Index Ventures, alongside Sequoia Capital, Slow Ventures, Coature, Neo, Amplify Partners, and other leading investors.
```
Comment (mgasner) [inline pages/blog/decade-of-data.mdx:15]: consider `has raised`

File: pages/blog/decade-of-data.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+![Dagster x Elementl](/posts/decade-of-data/elementl-investors.png)
+
+That’s not our only big news. We're also announcing our hosted product, Dagster Cloud, which unifies all of your data tools into a productive, seamless, enterprise-grade data platform. Early access to Dagster Cloud will be available in the coming weeks. [Sign up here](https://dagster.io/cloud) to be the first in line.
```
Comment (mgasner) [inline pages/blog/decade-of-data.mdx:19]: consider just `enterprise-grade`

File: pages/blog/decade-of-data.mdx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+Dagster has been built out in the open, but we’ve been quietly building Elementl, the company behind it, along the way.
+
+Today, I’m excited to announce that Elementl raised a $14M Series A led by Mike Volpi of Index Ventures, alongside Sequoia Capital, Slow Ventures, Coature, Neo, Amplify Partners, and other leading investors.
+
+![Dagster x Elementl](/posts/decade-of-data/elementl-investors.png)
+
+That’s not our only big news. We're also announcing our hosted product, Dagster Cloud, which unifies all of your data tools into a productive, seamless, enterprise-grade data platform. Early access to Dagster Cloud will be available in the coming weeks. [Sign up here](https://dagster.io/cloud) to be the first in line. 
+ 
+Over the next several weeks, we’re publishing a five-part series on the Dagster blog about why Elementl exists, why Dagster is our first project, and where things are going from here.
```
Comment (mgasner) [inline pages/blog/decade-of-data.mdx:21]: consider `we'll be publishing`

File: pages/blog/decade-of-data.mdx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+ 
+But developing high-quality, trusted data assets is a challenging engineering problem. Data is ingested from dozens––if not hundreds––of external sources, SaaS apps, and operational systems of record. Ripped from its original context, data must be integrated and transformed by practitioners into usable and meaningful forms. These data assets must be managed and served to downstream stakeholders and processes: business decision-makers, machine learning models, other data practitioners, and production applications.
+ 
+The process of transforming, managing, and making sense of data crosses personas and skillsets; involves many different types of tools and technologies; is riddled with organizational, operational, and technical complexity; and is critical to the success of every modern organization.
+
+We call this a data platform: the tools, services, and applications that store, transform, manage, and serve data outside its original source or system of record.
+ 
+Every organization has a data platform that manages its data assets. The only question is whether it is formally acknowledged and staffed or not.
```
Comment (mgasner) [inline pages/blog/decade-of-data.mdx:55]: consider a callout/drop quote for this (`>`)

File: pages/blog/decade-of-data.mdx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+<blockquote>The era of Big Data is over; the time of Big Complexity has come.</blockquote>
+
+Now that computation and storage at scale are possible, we need to ensure that data assets are productively used and efficiently managed. Today, our problem isn’t the physical processing of data at scale, but managing the complexity of the data platform. Practitioner productivity, data management, and other issues “higher in the stack” will dominate data infrastructure development and tooling for the next decade. The era of Big Data is over; the time of Big Complexity has come.
+
+
+Big Complexity can’t be wished away. It must be managed and tamed by making engineers, practitioners, and stakeholders more productive and efficient. We think this is one of the most consequential and formidable software engineering challenges of the next decade.  The benefits of productivity and taming complexity compound and scale, not only making existing processes more efficient, but entirely new, unimagined things possible. And that’s why Dagster exists. Our mission: to empower every organization to build a productive, scalable data platform.
```
Comment (mgasner) [inline pages/blog/decade-of-data.mdx:68]: consider `Our mission is` to align with the drop quote

File: pages/blog/decade-of-data.mdx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+
+<blockquote>Our mission is to empower every organization to build a productive, scalable data platform.</blockquote>
+
+# The Path to Impact: Data Orchestration
+
+Data orchestration is the beating heart of any organization’s data platform. Any practitioner putting a data asset into production must interact with the orchestrator because all data must come from somewhere and go somewhere. Orchestration invokes every data processing tool, and those tools, in turn, touch every storage system. Orchestration is the layer that interconnects all tools, data, practitioners, and stakeholders.
+
+
+<blockquote> Orchestration is the layer that interconnects all tools, data, practitioners, and stakeholders.
```
Comment (mgasner) [inline pages/blog/decade-of-data.mdx:78]: consider `all the tools... on the data platform`

File: pages/blog/decade-of-data.mdx (hunk 1 above)
Comment (schrockn) [reply pages/blog/decade-of-data.mdx:15]: deliberate choice to make timing a bit ambiguous

File: pages/blog/decade-of-data.mdx (hunk 5 above)
Comment (schrockn) [reply pages/blog/decade-of-data.mdx:68]: In the context of the paragraph I prefer the colon version

File: pages/blog/decade-of-data.mdx (hunk 6 above)
Comment (schrockn) [reply pages/blog/decade-of-data.mdx:78]: within the data platform?

File: pages/blog/decade-of-data.mdx (hunk 4 above)
Comment (schrockn) [reply pages/blog/decade-of-data.mdx:55]: which part, precisely

File: pages/blog/decade-of-data.mdx (hunk 3 above)
Comment (schrockn) [reply pages/blog/decade-of-data.mdx:21]: yup

**Pull request merged**